### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Eastsidegunn**
I luv u donggyu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
-# IdolGAN
\ No newline at end of file
+# IdolGAN ~~[adoːkẽꜜɴ]~~
+
```
Comment (vkehfdl1) [inline README.md:2]: Need more instruction of IdolGAN project. Like the purpose of this project, motivation, what we did and our further works. This abstract is so simple for IdolGAN main repository. Edit this pls.

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+We use [HiFaceGAN](https://github.com/Lotayou/Face-Renovation) and [KID-F Dataset](https://github.com/PCEO-AI-CLUB/KID-F).
+
+# Usage
+### Environment
+- Ubuntu/CentOS
```
Comment (vkehfdl1) [inline README.md]: delete CentOS because we didn't test it.

File: README.md [hunk 3]
```diff
@@ -1 +1,51 @@
...
+- PyTorch 1.0+
+- CUDA 10.1
+- python packages: opencv-python, tqdm, 
+- Data augmentation tool: [imgaug](https://imgaug.readthedocs.io/en/latest/source/installation.html#installation-in-pip)
+- [Face Recognition Toolkit](https://github.com/ageitgey/face_recognition) for evaluation
+- [tqdm](https://github.com/tqdm/tqdm) to make you less anxious when testing:)
+
```
Comment (vkehfdl1) [inline README.md:26]: Also, we miss our mediapipe for face detection.
Now we just need requirements for crop.py & degrad.py. 
Like tqdm, python, mediapipe, cv2, etc. Pls check that.

File: README.md [hunk 4]
```diff
@@ -1 +1,51 @@
...
+### Checkpoint
+We provide Model Checkpoint which trained from KID-F Dataset. We used train_dataset of KID-F for training, which takes 2 days on a GTX 1080 Ti. We upload the checkpoint on our [googledrive](https://drive.google.com/drive/folders/1GrZIofQc3uWFVWserxgPEO97DZEJYOuK?usp=sharing).
```
Comment (vkehfdl1) [inline README.md]: googledrive ==> Google Drive

File: README.md [hunk 5]
```diff
@@ -1 +1,51 @@
...
+
+### Configurations
+The configurations is stored in `options/config_hifacegan.py`, the options should be self-explanatory, but feel free to leave an issue anytime.
```
Comment (vkehfdl1) [inline README.md:35]: Maybe we can explain how to edit config_hifacegan.py for our KID-F dataset

File: README.md [hunk 6]
```diff
@@ -1 +1,51 @@
...
+
+### Training and Testing
+```
+python train.py            # A fool-proof training script
+python test.py             # Test on synthetic dataset
+python test_nogt.py        # Test on real-world images
+python two_source_test.py  # Visualization of Fig 5
+```
```
Comment (vkehfdl1) [inline README.md]: delete these line pls.

File: README.md [hunk 7]
```diff
@@ -1 +1,51 @@
...
+### Evaluation
+We used PSNR, SSIM, FID, LPIPS, IDD.
+Coming Soon!
+
+### Benchmark
+
+We use HiFaceGAN model. 
+We compared HiFaceGAN's performance about 300 K-pop Idol images with 5 Metrices(PSNR, SSIM, FID, LPIPS, IDD).
```
Comment (vkehfdl1) [inline README.md]: check grammar at grammarly again.

File: README.md [hunk 8]
```diff
@@ -1 +1,51 @@
...
+|            |       FFHQ     |     Idol Dataset (Ours)    |
+|:----------:|:--------------:|:--------------------------:|
+|      PSNR↑ |      31.82     |            32.49           |
+|      SSIM↑ |       0.72     |            0.758           |
+|     LPIPS↓ |      0.151     |            0.109           |
+|      FID↓  |      0.574     |            0.159           |
+|      IDD↓  |     0.00637    |           0.00496          |
+
+### License
+Don't do what you think. anything
```
Comment (vkehfdl1) [inline README.md]: LOL.
Just copy from KID-F dataset agreement.

**Review by vkehfdl1 — CHANGES_REQUESTED** [review]
Check your grammar and follow some instructions that I requested. Thank you for your hard work.

File: README.md (hunk 5 above)
Comment (Eastsidegunn) [reply README.md:35]: Hey DongGyu it's your Job. Thx :) @vkehfdl1

**Review by vkehfdl1 — APPROVED** [review]
Funny description LOL
I'll edit this later for us. Good

**Pull request merged**